apps/gamify/web-app/src/app/core/interceptors/auth.interceptor.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpHandler, HttpInterceptor, HttpRequest } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+
+  intercept(req: HttpRequest<any>, next: HttpHandler) {
+    if (req.url.includes('https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key=')) {
```

Comment:
Maybe you want to capture this url in a constants file in the future. But good enough for now.